services/httpd/handler.go

```diff
@@ -954,14 +955,17 @@ func parseSystemDiagnostics(d *diagnostics.Diagnostics) (map[string]interface{},
 }
 
 // httpError writes an error to the client in a standard format.
-func (h *Handler) httpError(w http.ResponseWriter, error string, code int) {
```

Comment:
I think `error` is a better parameter name. At the least, `errmsg` so that the parameter name describes that it's expecting the text of an error.